AnarchyExploitFixesFolia/src/main/java/me/xginko/aef/modules/illegals/items/IllegalItemModule.java

```diff
@@ -54,15 +54,19 @@ public abstract class IllegalItemModule extends AEFModule implements Listener {
     protected final AEFPermission bypassPermission;
     protected final IllegalHandling handling;
     protected final Set<Listener> optionalListeners;
-
+    private final boolean guiPluginsSupported;
     private final Cache<Class<? extends Event>, ExpiringSet<Object>> listenerCooldowns;
     private final Function<Class<? extends Event>, @PolyNull ExpiringSet<Object>> createIfAbsent;
 
     public IllegalItemModule(String configPath, AEFPermission bypassPermission) {
         super(configPath);
         this.bypassPermission = bypassPermission;
         this.optionalListeners = new HashSet<>();
-
+        this.guiPluginsSupported = config.getBoolean(configPath + ".gui-plugins-supported", false, """
+                Don't enable this unless you have a plugin on the menu gui.
```

Comment:
"Don't enable this unless you have a plugin on the menu gui." Is a confusing sentence. Change it to "Enable this if you have problems with the plugin removing items from chest gui's."